triplet/main.py

```diff
@@ -151,7 +145,13 @@ def convert_and_transform(img, transform):
         return img_a, img_p, img_n
 
     def __len__(self):
-        return self.triplets.shape[0]
+        try:
+            if self.train:
+                return self.triplets.shape[0]
+            else:
+                return self.matches.shape[0]
+        except:
+            pass
 
```

Comment:
@apaszke I had to hack this since `TripletPhototour` inherits from `Phototour` and for this reason the parent class doesn't have the train attribute which makes crash the program. Any suggestion?